### PR TITLE
feat(arby): add disabled option

### PR DESCRIPTION
**Pull request by reliveyy**
This PR adds 4 arby related command-line options:
* `--arby.disabled`
* `--arby.binance-api-key`
* `--arby.binance-api-secret`
* `--arby.margin`

### How to test?

```
bash setup.sh -b disabled-option --arby.disabled
bash setup.sh -b disabled-option --arby.binance-api-key abc --arby.binance-api-secret 123 --arby.margin 0.001
```

### CODE REVIEW

**Comment by raladev** [comment]
It seems like arby should be disabled as default because it is additional soft. WDYT @kilrau ?

**Comment by kilrau** [comment]
It should be **enabled** by default. It is additional software, just like lnd, connext etc but which is crucially needed by market makers to do their job, since xud-docker is *the* setup for market makers we want it to be enabled by default.

**Comment by kilrau** [comment]
And good job on improving PR descriptions with test instructions :+1: @reliveyy

**Comment by reliveyy** [comment]
> It should be **enabled** by default. It is additional software, just like lnd, connext etc but which is crucially needed by market makers to do their job, since xud-docker is _the_ setup for market makers we want it to be enabled by default.

FYI, the default value of `--arby.disabled` is **False**.

**Comment by kilrau** [comment]
> FYI, the default value of --arby.disabled is False.

Aware and that's how it should be. All good!

**Comment by reliveyy** [comment]
I agree that we need invalid/missing parameter warnings @raladev @kilrau

**Comment by kilrau** [comment]
> I agree that we need invalid/missing parameter warnings @raladev @kilrau

Sure but that applies to a bunch of other parameters too (like external bitcoind/litecoind). For this PR it's enough to ignore other arby parameters if `--arby.disabled` is set.

File: images/utils/launcher/config/simnet.conf [hunk 1]
```diff
@@ -37,3 +37,4 @@
 binance-api-key = "123"
 binance-api-secret = "321"
 margin = "0.04"
+disabled = false
```
Comment (raladev) [inline images/utils/launcher/config/simnet.conf:40]: all params should be commented out (#)

**Pull request merged**